packages/avatar/src/react/utils.js

```diff
@@ -12,12 +12,76 @@ export const getInitials = fullname => {
 }
 
 export const getColorByName = name => {
-  const initial = name[0].toUpperCase() || null
-  return colorByLetter[initial] || defaultColor
+  const stableRandLetter =
+    sha1(name)
+      .toUpperCase()
+      .replace(/[^A-Z]/, '')[0] || 'A'
+  return colorByLetter[stableRandLetter] || defaultColor
```

Comment:
I think `defaultColor` probably never gonna be used on this case, once we have `'A'` as fallback for `stableRandLetter`, If so maybe remove it?

Reply:
yes, good idea!